### DISCUSSION TITLE
Why crypto and NFTs are the worst that happened with us

### DISCUSSION THREAD

**Discussion by max-arnold** [rufuspollock/awesome-crypto-critique#87] [General]
Not sure if articles in Russian language are relevant for your colletion, but this monumental piece just came out 

https://hyperduh.medium.com/%D0%BD%D0%B5-%D0%B2%D0%B0%D1%88%D0%B8-%D0%B1%D1%80%D0%BE-%D0%BF%D0%BE%D1%87%D0%B5%D0%BC%D1%83-%D0%BA%D1%80%D0%B8%D0%BF%D1%82%D0%B0-%D0%B8-nft-%D1%85%D1%83%D0%B4%D1%88%D0%B5%D0%B5-%D1%87%D1%82%D0%BE-%D1%81-%D0%BD%D0%B0%D0%BC%D0%B8-%D1%81%D0%BB%D1%83%D1%87%D0%B8%D0%BB%D0%BE%D1%81%D1%8C-f8b57be3d550#7b85

Hopefully it will be translated by someone soon

**Comment by rufuspollock** [comment]
@max-arnold thanks - good find. do you want to take it on a spin through google translate at least to give us the intro?

**Comment by max-arnold** [comment]
Sorry, I do not have the energy to do that :(   I tried to wrap the link with [google translate](https://hyperduh-medium-com.translate.goog/%D0%BD%D0%B5-%D0%B2%D0%B0%D1%88%D0%B8-%D0%B1%D1%80%D0%BE-%D0%BF%D0%BE%D1%87%D0%B5%D0%BC%D1%83-%D0%BA%D1%80%D0%B8%D0%BF%D1%82%D0%B0-%D0%B8-nft-%D1%85%D1%83%D0%B4%D1%88%D0%B5%D0%B5-%D1%87%D1%82%D0%BE-%D1%81-%D0%BD%D0%B0%D0%BC%D0%B8-%D1%81%D0%BB%D1%83%D1%87%D0%B8%D0%BB%D0%BE%D1%81%D1%8C-f8b57be3d550?_x_tr_sl=ru&_x_tr_tl=en&_x_tr_hl=en-US&_x_tr_pto=wapp), but all it does is displays an empty page. I guess Medium is not a Google Translate friendly site...

So, it is up to you whether to publish that link as-is or close the issue, I'll be happy with both outcomes
